internal/server/v0/statvarsummary/golden/get_statvar_summary/females.json

```diff
@@ -337,31 +338,32 @@
           "maxValue": 72821
         },
         "Place": {
-          "placeCount": 172,
+          "placeCount": 15,
```

Comment:
The number get much smaller here

Reply:
I think that is because "Place" type will get mapped as a more specific type.

Reply:
Also the "HighSchoolDistrict" gets lower number which also occurs in some diff below.

Reply:
Not sure if you saw this, but responding inline the previous comment:

“From my recollection that gets tagged as SchoolDistrict (whose count
increased), which makes sense given the type preference order in
table_types.texproto“

Reply:
Sg. Just wonder is the previous specific type more accurate.

Reply:
Yea, totally not blocking but would be good to restore specific type later.